packages/taquito-tezbridge-wallet/src/taquito-tezbridge-wallet.ts

```diff
@@ -63,14 +63,9 @@ export class TezBridgeWallet implements WalletProvider {
     const { operation_id } = await tezbridge.request({
       method: 'inject_operations',
       operations: params.map(op => ({
-        ...this.removeFeeAndLimit(op),
+        op,
```

Comment:
I am not sure Tezbridge accepts those parameters. Did you test it?

Reply:
I have limit these changes to beacon.